prov/efa/test/efa_unit_test_info.c

```diff
@@ -99,6 +99,64 @@ void test_info_open_ep_with_api_1_1_info()
 	assert_int_equal(err, 0);
 }
 
+/**
+ * @brief test use_device_rdma behavior before and after API 1.18
+ */
+void test_info_open_ep_with_api_1_18_info()
```

Comment:
There are changes  to unit test that went into 1st commit.

Can you put all unit test changes in a separate commit?